repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py

```diff
@@ -208,54 +206,72 @@ def process(self):
                          ' to proceed the upgrade. Register your system with the'
                          ' subscription-manager tool and attach'
                          ' it to proper SKUs to be able to proceed the upgrade.'))
-            self.produce_error(error)
+            preparetransaction.produce_error(self, error)
             return
 
         # TODO: Find a better place where to run this (perhaps even gate this behind user prompt/question)
-        preparetransaction.call(['/usr/bin/dnf', 'clean', 'all'])
+        # FIXME: fails on insufficient permissions
```

Comment:
Insufficient permissions? When do you encounter that? Just out of curiosity

Reply:
@vinzenz selinux would forbid the access possibly?

Reply:
@vojtechsokol As I wrote, even when user is root, it can be hit that it will not be able to open a file for write because of selinux or another security technology.

Reply:
@pirat89 However in this case this problem can happen to virtually all subprocess calls. Is it necessary then to keep those FIXME comments? If yes, they should be added to all subprocess calls to be consistent...

Reply:
@vojtechsokol: nope. But maybe rather we should detect something like that later in the call function itself. Or by the guard funsction.